nixos/modules/services/misc/linux-enable-ir-emitter.nix

```diff
@@ -0,0 +1,48 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
+  cfg = config.services.linux-enable-ir-emitter;
+in
+{
+  options = {
+    services.linux-enable-ir-emitter = {
+      enable = mkEnableOption (mdDoc "Linux Enable IR Emitter");
+
+      package = mkPackageOptionMD pkgs "linux-enable-ir-emitter" {} // {
+        description = mdDoc ''
+          Package to use for the Linux Enable IR Emitter service.
```

Comment:
```suggestion
          Package to use for the Linux Enable IR Emitter service. May be useful for Howdy facial authentication.
```

Reply:
I'll add it to the `enable` text. Also we shouldn't necessarily enable the service as not all laptops need configuring for the IR cam to work.